free-1.md

```diff
@@ -10,34 +10,67 @@ _What need is met by your idea? Who are you helping? What is that person's
 them?_
 
+My idea helps people who are writing game rules to ensure that they are
+well-written without loopholes. Currently, game makers have to think hard about
+edge cases and not just try to figure out what they are, but how they should be
+handled so that they are most consistent with the rest of the rules. If they
+used a DSL stemming from my idea, they could be notified about what cases are
+left undecided, and the language could automatically help output their rules in
+clear English.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because the game maker can write their rules in the
+language which will be optimized for rule writing. Writing rules in the
+language will make rule-writing clearer because the syntax will be more
+optimized for rules than English is. Then the system can automatically check
+their rules for consistency and loopholes.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Often when I'm playing games I notice things that either don't really have a
+rule applied to them or the rule that does govern how something works doesn't
+make sense with the larger theme.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Building and verifying game rules.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The language will mostly be a text-based interface which is similar to a
+standard programming language. The user can define different types of game
+objects and how they interact. Programming is basically defining these game
+objects in one section and in another section explaining how different classes
+work together. There can be different layers of objects (so for a card game,
+the ace of hearts can be part of the hearts class and the aces class) and rules
+apply to each one.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The main non-syntax error would be that there is some case that the game
+designer missed. These could just be printed out when the program is run.
+If the language is accepted, rules printed in English will be output, and these
+rules are guaranteed to be unambiguous.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to define most games. Simple turn-based card and board games
+will obviously be easier than a complex real-time strategy online game, but it
+should be possible to express both. I can't think of anything in the domain
+that should be really impossible --- ideally, the system is flexible enough to
+define most rule sets.
```

Comment:
I'm not sure it's quite so straightforward to define most games. Actually, no, I think I've misunderstood your idea a bit.  I guess you are not trying to check also that the logic of the game makes any sense.  That is, you're aiming for the equivalent of a type-checker for game rules. This still seems like it might end up being a pretty hard problem in the general case, but I guess we'll see.